lib/Mojolicious/Plugin/OpenAPI.pm

```diff
@@ -253,6 +258,28 @@ sub _route_path {
   return $path;
 }
 
+sub _security_action {
+  my ($self, $security_settings) = @_;
+  my $security_cb = $self->_security_cb;
+
+  return sub {
+    my $c = shift;
+    my @cc
+      = map { my ($name, $config) = %$_; ($security_cb->{$name}, $config); } @$security_settings;
```

Comment:
`$config` is not a callback. In this case, `$name` is "dummy" and `$config` is `[]`. You don't get the "securityDefinitions" object. You get the data from `"security": [{"dummy": []}]`, meaning it will always (I think) be an empty array, except if you work with oauth2.